application/controllers/service_helper.py

```diff
@@ -4,85 +4,17 @@
 import structlog
 from flask import current_app
 
-from application.exceptions import RasError
+from application.exceptions import RasError, ServiceUnavailableException
 
 log = structlog.wrap_logger(logging.getLogger(__name__))
 
 
-def get_case_group(case_id):
-    """
-    Get case details from service
-    :param case_id: The case_id to search with
-    :return: case details
-    """
-
-    case_group = None
-    response = service_request(service="case-service", endpoint="cases", search_value=case_id)
-
-    if response.status_code == 200:
-        case = response.json()
-        case_group = case.get("caseGroup")
-    else:
-        log.error("Case not found", case_id=case_id)
-    return case_group
-
-
-def get_collection_exercise(collection_exercise_id):
-    """
-    Get collection exercise details from request
-    :param collection_exercise_id: The collection_exercise_id to search with
-    :return: collection_exercise
-    """
-
-    collection_exercise = None
-    response = service_request(
-        service="collectionexercise-service", endpoint="collectionexercises", search_value=collection_exercise_id
-    )
-
-    if response.status_code == 200:
-        collection_exercise = response.json()
-    else:
-        log.info("Collection Exercise not found", collection_exercise_id=collection_exercise_id)
-    return collection_exercise
-
-
-def get_survey_ref(survey_id):
+def get_survey_service_details(survey_id):
```

Comment:
It's a bit picky, but I think this should be either `get_survey_details` or even just `get_survey`